packages/core/src/lib/connection_manager.ts

```diff
@@ -38,32 +38,15 @@ export class ConnectionManager
 
   private currentActiveParallelDialCount = 0;
   private pendingPeerDialQueue: Array<PeerId> = [];
-  private online: boolean = false;
 
-  public isConnected(): boolean {
-    return this.online;
-  }
+  private isP2PNetworkConnected: boolean = false;
 
-  private toggleOnline(): void {
-    if (!this.online) {
-      this.online = true;
-      this.dispatchEvent(
-        new CustomEvent<boolean>(EConnectionStateEvents.CONNECTION_STATUS, {
-          detail: this.online
-        })
-      );
+  public isConnected(): boolean {
+    if (globalThis?.navigator && !globalThis?.navigator?.onLine) {
```

Comment:
This one is tricky. I made it that was as to comply with behavior of https://github.com/waku-org/js-waku/blob/409642d7d37135b96f61608d34a7c1af8abca407/packages/discovery/src/local-peer-cache/index.ts#L128

when if we run `localStorage` discovery in `nodejs` it won't fail tho it won't work

the thing is even though we don't support nodejs - there are some people running `js-waku` in such env